### PR TITLE
[SYTO-921] Video playlist design fixes

### PR DESCRIPTION
**Pull request by hkekeli**
https://jira.natgeo.com/browse/SYTO-921

Instructions:

git checkout 
npm install
gulp serve

What to test:

On a large screen, confirm the background of the component extends the full width of the browser while the content is centered in the viewport.

### CODE REVIEW

**Comment by sebasrgl** [comment]
@hkekeli Could you update this branch? Also, Could you validate with Kevin the video title font size, since it seems to me very tiny?

**Comment by hkekeli** [comment]
Updated. The font size change was specified in the AC.

File: src/modules/videoplaylist/VideoPlaylist.js [hunk 1]
```diff
@@ -235,25 +235,28 @@ class VideoPlaylist extends Component {
       return <VideoThumbnail key={index} wrapperClass={thumbClass} kickerType={kickerType} item={item} onClick={this.handleClick.bind(this, index)}/>
     });
     return(
-      <div ref="mainVideoContainer" className="mt3_video-playlist mt3_col-12 mt3_col-lg-12 mt3_bgcolor--black">
-        <div className="mt3_col-12 mt3_col-md-9 mt3_video-playlist__flex">
-          <div className="mt3_video-playlist__main-head">
-            <div className="mt3_video-playlist--heading mt3_video-playlist--heading--border">
-              <div className="mt3_video-playlist--heading__title mt3_color--white">{this.props.dataModel.heading}</div>
+
+      <div className="mt3_bgcolor--black">
```
Comment (arieldelfor) [inline src/modules/videoplaylist/VideoPlaylist.js]: please remove this block and place "mt3_bgcolor--black" class within the next wrapper.

At 1600px the background color should not overflow beyond the width of component.

i.e: `mt3_row mt3_video-playlist mt3_video-playlist__wrapper mt3_bgcolor--black`

**Comment by arieldelfor** [comment]
LGTM. Jira AC updated.
A new ticket will be created to handle the wide black background.

**Pull request merged**